libs/libcommon/tests/test_simple_cache.py

```diff
@@ -125,6 +127,52 @@ def assert_metric(http_status: HTTPStatus, error_code: Optional[str], kind: str,
     assert metric.total == total
 
 
+def test_upsert_response_retries() -> None:
+    kind = CACHE_KIND
+    dataset = DATASET_NAME
+    dataset_git_revision = REVISION_NAME
+    config = None
+    split = None
+    content = {"some": "content"}
+    job_params: JobParams = {"dataset": dataset, "config": config, "split": split, "revision": dataset_git_revision}
+    assert CacheTotalMetricDocument.objects().count() == 0
+    # insert for the first time
+    upsert_response_params(
+        kind=kind,
+        job_params=job_params,
+        content=content,
+        http_status=HTTPStatus.OK,
+    )
+    cached_entry_metadata = get_response_metadata(kind=kind, dataset=dataset, config=config, split=split)
+    assert cached_entry_metadata is not None
+    assert cached_entry_metadata["retries"] == 0
+    assert CacheTotalMetricDocument.objects().count() == 1
+    # insert for the second time
+    upsert_response_params(
+        kind=kind,
+        job_params=job_params,
+        content=content,
+        http_status=HTTPStatus.OK,
+    )
```

Comment:
OK. that means that if we force-refresh from the admin endpoint, it will increment the value too. I think it's ok.